integration-cli/docker_cli_events_test.go

```diff
@@ -146,9 +146,21 @@ func TestEventsImageUntagDelete(t *testing.T) {
 func TestEventsImagePull(t *testing.T) {
 	since := time.Now().Unix()
 
-	defer deleteImages("hello-world")
+	defer setupRegistry(t)()
+	repoName := fmt.Sprintf("%v/dockercli/hello-world", privateRegistryURL)
```

Comment:
This can be updated to be named busybox since it is actually a busybox image. "hello-world" is only used when pulling from the registry because it is a very small image.